front/src/applications/operationalStudies/types.ts

```diff
@@ -68,6 +68,14 @@ export type ManageTrainSchedulePathProperties = {
   incompatibleConstraints?: IncompatibleConstraints;
 };
 
+export type StdcmPathProperties = {
+  manchetteOperationalPoints?: OperationalPoint[];
+  geometry: NonNullable<PathProperties['geometry']>;
+  suggestedOperationalPoints: SuggestedOP[];
+  zones: NonNullable<PathProperties['zones']>;
+  operationalPoints: SuggestedOP[];
+};
```

Comment:
Move this in applications/stdcm/types.ts and remove `operationalPoints` property

```suggestion
export type StdcmPathProperties = {
  manchetteOperationalPoints?: OperationalPoint[];
  geometry: NonNullable<PathProperties['geometry']>;
  suggestedOperationalPoints: SuggestedOP[];
  zones: NonNullable<PathProperties['zones']>;
};
```